composer.json

```diff
@@ -5,7 +5,14 @@
     "description": "This package provides wrapper functionality for the Elasticsearch engine.",
     "require": {
         "ext-curl": "*",
-        "neos/flow": "~4.0 | dev-master"
+
+        "doctrine/annotations": "^1.0",
+        "doctrine/collections": "^1.0",
+        "doctrine/orm": "^2.0",
+        "neos/error-messages": "^4.0",
+        "neos/flow": "^4.0",
+        "neos/utility-arrays": "^4.0",
+        "neos/utility-objecthandling": "^4.0"
```

Comment:
Yes, I agree to myself and to @bwaidelich - but still: https://discuss.neos.io/t/composer-dependencies-declare-transitive-dependencies/1953 _Sigh_